src/commands/createNewProject/JavaScriptProjectCreator.ts

```diff
@@ -36,40 +34,50 @@ export class JavaScriptProjectCreator extends ScriptProjectCreatorBase {
 
     public getTasksJson(runtime: string): {} {
         let options: ITaskOptions | undefined;
+        // tslint:disable-next-line:no-any
+        const funcTask: any = {
+            label: funcHostTaskLabel,
+            identifier: funcHostTaskId,
```

Comment:
Are we going to get rid of `identifier`?

Reply:
Eventually, but it's not doing any harm so I'd rather just leave it for this release